tests/testthat/test_classif_mob.R

```diff
@@ -0,0 +1,37 @@
+context("classif.mob")
+
+test_that("autotest", {
+  # use a senseless logit model and partition with respect to all features
+  logit_ = function(y, x, start = NULL, weights = NULL, offset = NULL, ...) {
+    glm(y ~ 1, family = binomial, start = start, ...)
+  }
+  learner = LearnerClassifMob$new()
+  learner$param_set$values$rhs = " . "
+  learner$param_set$values$fit = logit_
+  learner$param_set$values$additional = list(maxit = 100)
+  learner$feature_types = c("logical", "integer", "numeric", "factor", "ordered")
+  learner$properties = c("twoclass", "weights")
+
+  # FIXME: this is from mlr3learners helpers
```

Comment:
We should move this to the autotest helpers in mlr3 with a short comment. Can you please open a PR there?

Reply:
done, mlr-org/mlr3#486